docs/logs/settings.md

```diff
@@ -0,0 +1,84 @@
+title: Sematext Logs Settings
+description: Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
+
+The settings include:
+
+  - inviting new users to your App
+  - [authorizing public IPs to send data to your app via syslog](../logs/authorizing-ips-for-syslog)
+  - adjusting data retention time, daily volume, and limits
+  - [changing your app's plan](../logs/faq/#plans-prices)
+  - checking how much data is, or has been shipped to your App
+
+  ![Sematext Logs App Settings](https://sematext.com/docs/images/logs/logsene-app-settings.png)
+
+
+## Side Navigation
+
+The persistent region on the left that can be collapsed, is used to easily switch across monitoring, log management, transaction tracing, real user monitoring, and other user & team features. All side navigation tabs are selectable. If they have chevrons, it indicates that the top level section is collapsible and contains additional subitems. The subitems will let you explore all your Apps, or choose a particular App.
+
+Side navigation top level sections for infrastructure and application performance monitoring, log management, transaction tracing, and real user monitoring:
+
+- Dashboards
+- Infrastructure
+- Monitoring
+- Logs
+- Correlations
+- Alerts and Events
+- Integrations
+
+Side navigation sections for user and team features:
+
+- Invites
+- Team
+- Account 
+
+![](../images/guide/logs/side-navigation.png)
+
+## App & Dashboard Selector
+
+The App & Dashboard dropdown selector lets you **choose any Logs or Monitoring App** as well as any **Dashboards and Reports** that you have generated from those Apps. Infrastructure views such as AppMap, NetMap, Servers, and Containers are also available for selection, and together provide seamless switching and navigation between logs, application and infrastructure monitoring, which is essential when rushing to fix production issues.
+
+## App Actions
+
+Right next to the App & Dashboard Selector is the **App Actions dropdown menu** and **horizontal ellipsis icon**. 
+
+The App Actions dropdown menu allows you to **quickly switch between your Dashboards, Logs and Monitoring Apps**. While the horizontal elipsis icon shows your App options and lets you open various app specific functions, like instructions to **install monitors and start sending metrics, app settings, alert rules, heartbeat alerts, as well as ability to connect two different apps, invite team members, and transfer app ownership**.
```

Comment:
Generalize?
apps => Apps
app => App

Reply:
Removed _logs and monitoring_ and only mentioning _Apps_. And fixed capitalization.